src/App.tsx

```diff
@@ -1,166 +1,24 @@
-import { Loader } from './components/Loader';
-
+import { Routes, Route, Navigate } from 'react-router-dom';
```

Comment:
The task description specifies using `HashRouter` instead of the default `BrowserRouter` to ensure links can be shared on GitHub pages. Consider wrapping the `Routes` component with `HashRouter`.